src/components/data-management/ProjectDetails.jsx

```diff
@@ -256,19 +255,6 @@ const ProjectDetails = ({ width, height }) => {
         </div>
       );
     }
-
-    if (status === UploadStatus.DATA_MISSING) {
```

Comment:
why do we remove the `DATA_MISSING` status?

Reply:
It wasn't used any more with the data. However, it might be related to this https://biomage.atlassian.net/browse/BIOMAGE-1049

Reply:
by the fact that it might be related to 1049, do you mean that we might want to keep this status?

Reply:
After further investigation, keeping this piece of code does not show any controls for uploading individual files as required in BIOMAGE-1049, so I'll remove this code.